.travis.yml

```diff
@@ -1,5 +1,8 @@
 language: java
 
+jdk:
+  - oraclejdk8
```

Comment:
The top level pom forces 1.6:

https://github.com/prometheus/client_java/blob/master/pom.xml#L63

and only the vert.x one allows 1.8